rotkehlchen/tests/data/pnl_debug.json

```diff
@@ -0,0 +1,102 @@
+        "amount": "3.55448345",
+        "usd_value": "24455.415502078435"
+      },
+      "event_type": "receive",
+      "event_subtype": null,
+      "location_label": "0x19e4057A38a730be37c4DB690b103267AAE1d75d",
+      "notes": "Receive 3.55448345 ETH 0xaBEA9132b05A70803a4E85094fD0e1800777fBEF -> 0x19e4057A38a730be37c4D690b103267AAE1d75d",
+      "counterparty": "0xaBEA9132b05A70803a4E85094fD0e1800777fBEF"
+    },
+    {
+      "identifier": 2,
+      "event_identifier": "0xa9905f5eaa664a53e6513f7ba2147dcebc3e54d4062df9df1925116b6a220014",
+      "sequence_index": 0,
+      "timestamp": 1651259834,
+      "location": "blockchain",
+      "asset": "ETH",
+      "balance": {
+        "amount": "0.009",
+        "usd_value": "33.85395142596176076"
+      },
+      "event_type": "spend",
+      "event_subtype": "fee",
+      "location_label": "0x19e4057A38a730be37c4DB690b103267AAE1d75d",
+      "notes": "Burned 0.001367993179812 ETH in gas from 0x19e4057A38a730be37c4DB690b103267AAE1d75d",
+      "counterparty": "gas"
+    }
+  ],
+  "settings": {
+    "have_premium": false,
+    "version": 33,
+    "last_write_ts": 1654528773,
+    "premium_should_sync": false,
+    "include_crypto2crypto": true,
+    "last_data_upload_ts": 0,
+    "ui_floating_precision": 2,
+    "taxfree_after_period": 31536000,
+    "balance_save_frequency": 24,
+    "include_gas_costs": true,
+    "eth_rpc_endpoint": "http://localhost:8545",
+    "ksm_rpc_endpoint": "http://localhost:9933",
+    "dot_rpc_endpoint": "",
+    "main_currency": "USD",
+    "date_display_format": "%d/%m/%Y %H:%M:%S %Z",
+    "last_balance_save": 0,
+    "submit_usage_analytics": true,
+    "active_modules": [
+      "uniswap",
+      "eth2",
+      "aave",
+      "loopring",
+      "balancer",
+      "yearn_vaults_v2",
+      "makerdao_vaults",
+      "compound",
+      "liquity",
+      "pickle_finance",
+      "nfts",
+      "sushiswap"
+    ],
+    "cost_basis_method": "fifo",
+    "frontend_settings": "{\"defi_setup_done\":false,\"timeframe_setting\":\"REMEMBER\",\"visible_timeframes\":[\"All\",\"1Y\",\"3M\",\"1M\",\"2W\",\"1W\"],\"last_known_timeframe\":\"2W\",\"query_period\":5,\"profit_loss_report_period\":{\"year\":\"2022\",\"quarter\":\"ALL\"},\"thousand_separator\":\",\",\"decimal_separator\":\".\",\"currency_location\":\"after\",\"refresh_period\":-1,\"explorers\":{},\"items_per_page\":10,\"amount_rounding_mode\":0,\"value_rounding_mode\":1,\"dark_mode_enabled\":false,\"light_theme\":{\"primary\":\"#7e4a3b\",\"accent\":\"#e45325\",\"graph\":\"#96DFD2\"},\"dark_theme\":{\"primary\":\"#ff5722\",\"accent\":\"#ff8a50\",\"graph\":\"#E96930\"},\"graph_zero_based\":false,\"nfts_in_net_value\":true,\"dashboard_tables_visible_columns\":{\"ASSETS\":[\"percentage_of_total_net_value\"],\"LIABILITIES\":[\"percentage_of_total_net_value\"],\"NFT\":[\"percentage_of_total_net_value\"]},\"date_input_format\":\"%d/%m/%Y %H:%M:%S\",\"version_update_check_frequency\":24,\"enable_ens\":true}",
```

Comment:
hmmm serialized json should not contain escape characters. Something is wrong here